src/Centreon/Infrastructure/Gorgone/CommandRepositoryAPI.php

```diff
@@ -95,7 +95,15 @@ public function send(CommandInterface $command): string
             }
             $jsonResponse = json_decode($response->getContent(), true);
             if (!array_key_exists('token', $jsonResponse)) {
-                throw new \Exception('Token not found');
+                $exceptionMessage = 'Token not found';
+                if (array_key_exists('message', $jsonResponse)) {
+                    if ($jsonResponse['message'] === 'Method not implemented') {
+                        $exceptionMessage = 'The Autodisco module of Gorgone is not loaded';
```

Comment:
```suggestion
                        $exceptionMessage = 'The Autodiscovery module of Gorgone is not loaded';
```

Reply:
Don't why there is span, my suggestion was : The "autodiscovery" module of Gorgone is not loaded